inspirehep/modules/citations/receivers.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+
+from invenio_records.signals import after_record_insert, after_record_update,  before_record_index
+
+from .tasks import update_citation_count_for_records
+from .utils import calculate_citation_count
+
+
+@after_record_insert.connect
+def catch_citations_insert(sender, *args, **kwargs):
+
```

Comment:
Instead of the empty line can you add a docstring with a small explanation of the function? :)